assets/src/edit-story/components/canvas/singleSelectionMovable.js

```diff
@@ -253,6 +268,9 @@ function SingleSelectionMovable({ selectedElement, targetEl, pushEvent }) {
         setTransformStyle(target);
       }}
       onResizeEnd={({ target }) => {
+        if (wasElementDeleted(target)) {
+          return;
+        }
```

Comment:
Hm, is that correct? If I resize it and it happens to lie only outside the canvas, does that indicate intent to delete? I would not personally expect that to happen, nor use that as a way deleting an element.

Reply:
I definitely think it's an edge case, but I think this still makes sense to delete the element. Otherwise it will just live outside of the page without anyone remembering about it.

Reply:
Yes, that's expected, see also https://github.com/google/web-stories-wp/issues/445#issuecomment-638883310